### PR TITLE
[Circletracer] add dependency to package

### PR DESCRIPTION
**Pull request by CODe1995**
Add flatbuffers and dependency files.

ONE-vscode-DCO-1.0-Signed-off-by: Seunghyun Kang <code.57x53@gmail.com>

### CODE REVIEW

**Comment by CODe1995** [comment]
from draft #179
from issue #66.

File: package-lock.json [hunk 1]
```diff
@@ -1,8 +1,2853 @@
 {
   "name": "one-vscode",
   "version": "0.0.1",
-  "lockfileVersion": 1,
+  "lockfileVersion": 2,
```
Comment (seanshpark) [inline package-lock.json:4]: why is this changed?

Comment (CODe1995) [reply package-lock.json:4]: [lockfileVersion]
1: The lockfile version used by npm v5 and v6.
2: The lockfile version used by npm v7, which is backwards compatible to v1 lockfiles.

What version of npm are you using? I'm using v7.12.1

Comment (seanshpark) [reply package-lock.json:4]:
```
$ npm --version
6.14.10
```

Comment (CODe1995) [reply package-lock.json:4]: Thanks for checking😀
Even if npm is version 6, `lockfileversion` 2 is [compatible](https://stackoverflow.com/questions/66150813/is-lockfileversion-2-in-package-lock-json-from-npm-7-compatible-with-older-ve).
`lockfileversion` is an option that is automatically converted when using npm v7.
How about using `lockfileversion` to 2?

Comment (seanshpark) [reply package-lock.json:4]: As I'm not used to `npm` domain, I don't know it's OK or not.
I'll just follow your words.

**Comment by seanshpark** [comment]
Please resolve check-format failure.

**Comment by CODe1995** [comment]
> Please resolve check-format failure.

Checking `format.patch`, there is no problem other than adding the package contents while adding flatbuffers. Are there any other issues I'm missing out on?

**Comment by seanshpark** [comment]
Please check `Details` link and look at the format.patch file

**Comment by CODe1995** [comment]
> Please check `Details` link and look at the format.patch file

I confirmed with the team that there was nothing wrong with npm7. Perhaps it is because the npm version of the format checker is 6.

**Comment by llFreetimell** [comment]
Interesting... the `format.patch` includes reverting all the changes in this PR @.@ (Except `package.json`)

**Comment by seanshpark** [comment]
This is script to run check format in CI
```
  check-format:
    runs-on: ubuntu-20.04

    steps:
      - uses: actions/checkout@v2
      - uses: actions/setup-node@v2
        with:
          node-version: '12'
```
node version 12 be the reason?

**Comment by seanshpark** [comment]
I've reopened #21

**Comment by CODe1995** [comment]
If the npm version is lower, it is recommended to manually update it because there are security issues. In general, there is an npm version that is automatically installed for each nodejs, but we update it manually.

There are two workarounds.

1. Update the format checker's npm
```npm i -g npm```

2. I lower the npm version to version 6 and modify the code as required by the format checker.

what would be better?

**Comment by seanshpark** [comment]
> I lower the npm version to version 6 and modify the code as required by the format checker.
> what would be better?

As to land this change, this would be faster :)

**Comment by CODe1995** [comment]
> As to land this change, this would be faster :)

Done. Thank you :)

**Review by seanshpark — APPROVED** [review]
LGTM thank you!

**Review by llFreetimell — APPROVED** [review]
LGTM
